source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,116 @@
+.. _swupd_mixer_integration:
+
+Adding custom packages and bundles
+##################################
+
+
+
+Prerequisites
+=============
+
+To use the new swupd mixer feature, you'll need a recent image of Clear
+Linux OS for Intel Architecture with the following bundle installed. If you
+don't have it already, you can add it with the :command:`swupd bundle-add`
+command like so::
+
+  # swupd bundle-add mixer
+
+What is it
+================
+The software updater now has native integration with the mixer tool. One of the
+highest requests for Clear Linux OS for Intel Architecture was the ability for
+users to add their own custom content to their OS. Mixer initially enabled this,
+but it required users to spin their own full OS, and be the OSV for it making
+all the releases and update content.
+
+With this new feature, users can now add custom content ``without`` having to
+compose the entire OS themselves. For example, a user may want a package that
+does not exist in our OS yet. Creating an entire OS just to have a single package
+is doable, but not the correct way to solve the problem. Instead, users can now
+add custom packages and bundles using the :file:`add-pkg.sh` script, and simply
+calling :command:`swupd update && swupd bundle add <new_bundle>` to add just
+their new content. The regular upstream content will still be pulled from the
+normal URL, while local content will be added from a local update store. *Note*
+that while this feature is intended to enable additive, unique content, it is
+still possible to add non-unique packages, i.e replacing something already provided
+by upstream with a local version. However that is not guaranteed to always work, thus
+the updater will warn the user if conflicts are found before creating the local content.
```

Comment:
I'm still not clear if mix or upstream content takes priority and it should be mentioned here.